packages/controller-utils/package.json

```diff
@@ -60,6 +60,7 @@
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
     "@types/jest": "^27.4.1",
+    "bignumber.js": "^4.1.0",
```

Comment:
@legobeat Should this have been added to `dependencies` instead of `devDependencies`? It seems like it's being used in a public file.

Reply:
I think that `bignumber.js` is only used to pull the `BigNumber` type, because `bn.js` is used for the actual runtime conversion implementation